### PR TITLE
[12.0][Mig] web_searchbar_full_width

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: web_searchbar_full_width/README.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
+====================
+Full width searchbar
```
Comment (nikul-serpentcs) [inline web_searchbar_full_width/README.rst:2]: @bizzappdev Two Titles is Different?

Second: https://github.com/OCA/web/pull/1218/files#diff-b3dd890285fb4d4e41c8ae29c21f33a2R29

File: web_searchbar_full_width/views/templates.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <template id="assets_backend" name="web_searchbar_full_width assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <link rel="stylesheet" href="/web_searchbar_full_width/static/src/css/web_searchbar_full_width.scss"/>
```
Comment (nikul-serpentcs) [inline web_searchbar_full_width/views/templates.xml]: replace `css`  directory to `scss`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor change

**Comment by bizzappdev** [comment]
requested changes are done

File: web_searchbar_full_width/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Full width searchbar",
+    "version": "12.0.1.0.0",
+    "author": "Therp BV, "
```
Comment (sudhir-erpharbor) [inline web_searchbar_full_width/__manifest__.py:7]: add website key:
website': "http://github.com/OCA/web"

File: web_searchbar_full_width/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+              "Odoo Community Association (OCA), "
+              "Siddharth Bhalgami",
+    "license": "AGPL-3",
+    "category": "Tools",
```
Comment (sudhir-erpharbor) [inline web_searchbar_full_width/__manifest__.py:12]: category = web

File: web_searchbar_full_width/README.rst [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+   :target: https://runbot.odoo-community.org/runbot/162/12.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/web/issues/new?body=module:%20web_searchbar_full_width%0Aversion:%2012.0-mig-web_searchbar_full_width-krj-bizz%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (sudhir-erpharbor) [inline web_searchbar_full_width/README.rst]: version should be 12.0 in the feedback url

**Review by sudhir-erpharbor — COMMENTED** [review]
Minor changes

File: web_searchbar_full_width/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```
Comment (sudhir-erpharbor) [inline web_searchbar_full_width/__manifest__.py]: Link is wrong:

License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: web_searchbar_full_width/__manifest__.py (hunk 4 above)
Comment (bizzappdev) [reply web_searchbar_full_width/__manifest__.py:12]: IMO category will remain tools as this module is not adding any new functionality to the web

**Review by nikul-serpentcs — APPROVED** [review]
Code review LGTM :+1:

**Review by sudhir-erpharbor — APPROVED** [review]
LGTM

**Pull request merged**